packages/compass-user-data/src/user-data.ts

```diff
@@ -333,3 +332,150 @@ export class FileUserData<T extends z.Schema> extends IUserData<T> {
     return await this.readOne(id);
   }
 }
+
+// TODO: update endpoints to reflect the merged api endpoints https://jira.mongodb.org/browse/CLOUDP-329716
+export class AtlasUserData<T extends z.Schema> extends IUserData<T> {
+  private readonly authenticatedFetch;
+  private orgId: string = '';
+  private groupId: string = '';
```

Comment:
We use `projectId` instead of `groupId` in Compass. "Groups" are a legacy name for projects:
Some more info: https://www.mongodb.com/docs/atlas/api/atlas-admin-api-ref/#project-id